pkgs/development/python-modules/mozilla-django-oidc/default.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, setuptools
+, django
+, josepy
+, requests
+, cryptography
+, djangorestframework
+}:
+
+buildPythonPackage rec {
+  pname = "mozilla-django-oidc";
+  version = "4.0.1";
+  pyproject = true;
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "mozilla";
+    repo = "mozilla-django-oidc";
+    rev = version;
+    hash = "sha256-72F1aLLIId+YClTrpOz3bL8LSq6ZhZjjtv8V/GJGkqs=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    djangorestframework
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    (
+    export PYTHONPATH=.:$PYTHONPATH
+    export DJANGO_SETTINGS_MODULE=tests.settings
+    django-admin test
+    )
```

```suggestion
    export PYTHONPATH=.:$PYTHONPATH
    export DJANGO_SETTINGS_MODULE=tests.settings
    django-admin test
```
we don't need to worry about leaking those envs and subshells might mask exit codes if used incorrectly

or we just set the envs for this command

```suggestion
    PYTHONPATH=.:$PYTHONPATH DJANGO_SETTINGS_MODULE=tests.settings django-admin test
```